src/lib/triggers.js

```diff
@@ -172,9 +168,22 @@ class Triggers {
 							this.fire(trigger);
 						}, parseInt(condition.value, 10) * 1000);
 						break;
+					case 'chat-opened-by-visitor':
+						const openFunc = () => {
+							const { user } = store.state;
+							if (user) {
+								if (trigger.runOnce) { store.off('change', openFunc); }
+								setTimeout(() => {
+									this.fire(trigger).then(console.log).catch(console.error).finally(console.debug);
```

Comment:
🤔

Reply:
console.log is necesseary?

Reply:
console.log is forgotten code, setTimeout was unfortunately needed for it to work

Reply:
forget that, it's useless